api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -570,6 +570,12 @@ type HostedClusterSpec struct {
 	// +optional
 	Configuration *ClusterConfiguration `json:"configuration,omitempty"`
 
+	// operatorConfiguration specifies configuration for individual OCP operators in the cluster.
+	//
+	// +optional
+	// +openshift:enable:FeatureGate=ClusterVersionOperatorConfiguration
```

Comment:
please include this gate within api/hypershift/v1beta1/featuregates/ yaml files before running make update